src/env/node.js

```diff
@@ -11,11 +12,58 @@ import { globSync } from 'glob';
 import format from "../format-console.js";
 import { getType } from '../util.js';
 
-// Set up environment for Node
+// Recursively traverse a subtree starting from `node` and make (only) groups of tests collapsible
+function makeCollapsible (node) {
+	if (node.tests?.length) {
+		node.collapsed = node.test.level === 0 ? false : true; // the root is expanded by default
```

Comment:
I don't think it should be. The beauty of this mode  is that the default view can be a single line.

Reply:
Make sense. Fixed. Thanks.